### ISSUE TITLE
Generated iframes for Focus elements are missing a title

### ISSUE THREAD

**Issue by jolidog** [mautic/mautic#14845]
### Mautic Series

5.2.x series

### Mautic installed version

5.2

### Way of installing

Other (please specify if relevant in the bug report)

### PHP version

8.1.29

### What happened?

Enabled a focus element for a site that contains a basic newsletter form.
The iframe that is injected by the generated JS from mautic does not a title attribute which it needs for accessibility reasons.

Suggest we use the title of the focus element as the title of the iframe attribute.

### How can we reproduce this issue?

Step 1: Enabled a focus element for a site that contains a basic newsletter form.
Step 2: Check the website where the focus element appears, search the DOM for the iframe
Step 3: Confirm it has an empty title

### Relevant log output

```shell

```

### Code of Conduct

- [x] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by RCheesley** [comment]
Thanks for the report @jolidog - is this something you might be up for helping to fix?
